pkg/tanka/parse.go

```diff
@@ -44,12 +44,24 @@ func parse(dir string, opts *options) (*ParseResult, error) {
 		return nil, errors.Wrap(err, "reconciling")
 	}
 
+	if opts.applyLabels {
+		applyLabels(rec, env)
+	}
+
 	return &ParseResult{
 		Resources: rec,
 		Env:       env,
 	}, nil
 }
 
+func applyLabels(state manifest.List, env *v1alpha1.Config) {
```

Comment:
This does not belong here, it should be part of `Reconcile()` which is meant to transform resources

Reply:
Also I'd prefer not to have functions that modify their input variables. Given this is fairly short, just put it into `Reconcile`